Visualization/AFW_Display_Demo.ipynb

```diff
@@ -0,0 +1,323 @@
+    "\n",
+    "Once we define a string that contains the data directory, we start the `Butler` instance using the `lsst.daf.persistence` library alias `dafPersist` and its `Butler` class. The `Butler` object is initialized with a string containing the data directory we wish to access. Running the cell may take a few moments."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "datadir = \"/project/shared/data/Twinkles_subset/output_data_v2\"   #our data directory containing the Twinkles data organized as Butler expects\n",
+    "butler = dafPersist.Butler(datadir)                               #create an instance of the Butler, which we call `butler`, with access to our data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the `Butler` instance now generated using our data directory, we can retrieve the desired calibrated exposure by telling the butler which filter, raft, sensor, and visit we wish to view. To do this, we definie dictionary with the required information."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Grab a calexp of interest\n",
+    "dataId = {'filter': 'r', 'raft': '2,2', 'sensor': '1,1', 'visit': 235}          #Define a dictionary with the filter, raft, sensor, and visit we wish to view\n",
+    "calexp = butler.get('calexp', **dataId)                                         #retrieve the data using the `butler` instance and its function `get()`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **Step 3) Use AFWDisplay to Visualize the Image**\n",
+    "\n",
+    "Now, with a `Butler` instance defined and a calibrated exposure retrieved, we can use [`lsst.afw.display`](https://github.com/lsst/afw/tree/master/python/lsst/afw/display) to visualize the data.  The next task is to let AFWDisplay know that we want it to enroll `matplotlib` as our default display backend. To do this, we use the `setDefaultBackend()` function. Remember that we made an alias to `lsst.afw.display` called `afwDisplay`, so we'll use that to call `setDefaultBackend()`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "afwDisplay.setDefaultBackend('matplotlib')  # Use lsst.afw.display with the matplotlib backend"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are now set to display the image. To do this, we:\n",
+    "\n",
+    "* First create a `matplotlib.pyplot` figure using `plt.figure()` -- this will be familiar to anyone with experience using `matplotlib`.\n",
+    "* Then create an alias to the `lsst.afw.display.Display` method that will allow us to display the data to the screen.  This alias will be called `afw_display`.\n",
```

Comment:
I think that what you are actually doing is using the `afw_display` variable to persist the `lsst.afw.display.interface.Display` object that is returned by `lsst.afw.display.Display`.